skyportal/handlers/api/annotation.py

```diff
@@ -354,11 +432,16 @@ def put(self, associated_resource_type, resource_id, annotation_id):
 
         self.verify_and_commit()
 
-        if a.obj.id:  # comment on object, or object related resources
+        if a.obj.id:  # annotation on object, or object related resources
```

Comment:
Probably better the check the data type

Reply:
I want to also allow a spectrum annotation to update the source page. But a non-object commentable like CGN would not do that.